SimTKcommon/src/ParallelExecutor.cpp

```diff
@@ -213,6 +244,10 @@ int ParallelExecutor::getNumProcessors() {
 #endif
 }
 
+int ParallelExecutor::getNumPhysicalProcessors(){
+  return std::thread::hardware_concurrency();
```

Comment:
Why is this called `getNumPhysicalProcessors()` if it uses `hardware_concurrency()`? The doc for that says something about "number of thread contexts" and specifically says it isn't necessarily the "actual number" of processors. An alternative would be to use platform-specific methods that could obtain the true number of cores vs. the number of hyperthreads. It's not clear to me what you are trying to achieve here.